src_Core/CPU/CPU_Decode_C.bsv

```diff
@@ -341,7 +340,7 @@ function Tuple2 #(Bool, Instr) fv_decode_C_SQSP (MISA  misa,  Bit #(2)  xl, Inst
       Bool is_legal = ((misa.c == 1'b1)
                        && (op == opcode_C2)
                        && (funct3 == funct3_C_SQSP)
-                       && (xl == misa_mxl_128));
+                       && (xl == misa_mxl_128 || (xl == misa_mxl_64 && cap_enc)));
```

Comment:
Style (repeated a few times)

Reply:
I guess wrap and indent? I think I've missed quite a few extra conditions on these actually

Reply:
In fact, I think it makes sense to scrap mxl_128 entirely from these files.

Reply:
and parens